nautobot_ssot/integrations/infoblox/utils/diffsync.py

```diff
@@ -96,3 +96,20 @@ def get_default_custom_fields(cf_contenttype: ContentType) -> dict:
             if customfield.key not in default_cfs:
                 default_cfs[customfield.key] = None
     return default_cfs
+
+
+def map_network_view_to_namespace(network_view: str) -> str:
```

Comment:
What's the purpose for this? If it's only one mapping it seems a lil silly to have a full method for it. Are we expecting to add more mappings here?

Reply:
We need to do mapping in multiple places across multiple files. The alternative would be to have `if` conditional and hardcoding names in all of these places. This also makes it easy to extend in the future if we decide to allow remapping of the non-default network views.